application/helpers/export_helper.php

```diff
@@ -1406,6 +1406,29 @@ function quexml_set_default_value(&$element, $iResponseID, $qid, $iSurveyID, $fi
     }
 }
 
+/**
+ * Format defaultValue of Date/Time questions according to question date format
+ *
+ * @param mixed $element DOM element with the date to change
+ * @param int $qid The qid of the question
+ * @param int $iSurveyID The survey id
+ */
+function quexml_reformat_date(&$element, $qid, $iSurveyID)
+{
+    // Retrieve date format from the question
+    $questionAttributes = QuestionAttribute::model()->getQuestionAttributes($qid);
+    $dateformat = getDateFormatDataForQID($questionAttributes, $iSurveyID)['phpdate'];
```

Comment:
I don't like to see ` function()['key']` seems unclear to me.

Reply:
Updated! Thanks for the comment!